Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline{
                         stash includes: 'build-doxygen/doc_doxygen/html/**', name: 'doxydocs'
 
                         // get doxygen warnings
-                        warnings canComputeNew: false, canResolveRelativePaths: false, categoriesPattern: '', defaultEncoding: '', excludePattern: '.*README.*', healthy: '', includePattern: '', messagesPattern: '', parserConfigurations: [[parserName: 'Doxygen', pattern: 'build-doxygen/DoxygenWarningLog.txt']], unHealthy: '', failedTotalAll: '0'
+                        recordIssues filters: [excludeFile('.*README.*')], tools: [doxygen(pattern: 'build-doxygen/DoxygenWarningLog.txt')]
```

Comment:
excludeFile should be either excludeCategory or excludeType

Reply:
probably an `unstableTotalAll: 1` is also missing

Reply:
ok, seems like the filter already works. the unstable result should probably be testet

Reply:
> probably an `unstableTotalAll: 1` is also missing

Are you sure? Before it was set to zero, which in warnings-ng is the implicit default.

Reply:
could you just try what happens if you remove some comment, so enforce a doxygen warning?

Reply:
push